drv/lpc55-sprot-server/src/main.rs

```diff
@@ -78,7 +77,7 @@ fn configure_spi() -> Io {
     let gpio_driver = GPIO.get_task_id();
     setup_pins(gpio_driver).unwrap_lite();
     let gpio = drv_lpc55_gpio_api::Pins::from(gpio_driver);
-    // TODO: It should never happen but, initialization should be able to deal
+    // It should never happen but, initialization should be able to deal
```

Comment:
Not sure why we're removing the TODO if this isn't fixed.

Reply:
Also, it absolutely can happen -- all it would take is this task crashing during a transaction.

Reply:
@cbiffle @lzrd This can happen, but I'm not sure there's actually anything that needs doing with the current structure of the code. Either `next` will interrupt on `ssa`(CSn) and it will result in a flow error, or the SP will timeout and retry.  I think the comment should just be removed altogether at this point.

Reply:
I went ahead and deleted the comment. We can re-add it or open an issue if necessary, but I don't see what's actionable from it at the moment.